Tests/SwiftDocCTests/Model/SemaToRenderNodeTests.swift

```diff
@@ -2604,29 +2576,89 @@ Document
     
     /// Ensures we render our supported asides from symbol-graph content correctly, whether as a blockquote or as a list item.
     func testRenderAsides() async throws {
-        let asidesSGFURL = Bundle.module.url(
-            forResource: "Asides.symbols", withExtension: "json", subdirectory: "Test Resources")!
-        let (_, _, context) = try await testBundleAndContext(copying: "LegacyBundle_DoNotUseInNewTests", excludingPaths: []) { url in
-            try? FileManager.default.copyItem(at: asidesSGFURL, to: url.appendingPathComponent("Asides.symbols.json"))
-        }
-        
-        // Both of these symbols have the same content; one just has its asides as list items and the other has blockquotes.
-        let testReference: (ResolvedTopicReference) throws -> () = { myFuncReference in
+        let asidesSGFURL = Bundle.module.url(forResource: "Asides.symbols", withExtension: "json", subdirectory: "Test Resources")!
+        let catalog = Folder(name: "unit-test.docc", content: [
+            CopyOfFile(original: asidesSGFURL, newName: "Asides.symbols.json"),
+            TextFile(name: "Extension.md", utf8Content: """
+            # Some documentation
+            """),
+        ])
+
+        let (_, context) = try await loadBundle(catalog: catalog)
+
+        func testReference(
+            myFuncReference: ResolvedTopicReference,
+            expectedAsides: [RenderBlockContent.Aside],
+            file: StaticString = #filePath,
+            line: UInt = #line
+        ) throws {
             let node = try context.entity(with: myFuncReference)
             let symbol = node.semantic as! Symbol
             
             var translator = RenderNodeTranslator(context: context, identifier: node.reference)
             let renderNode = translator.visit(symbol) as! RenderNode
-            let asides = try XCTUnwrap(renderNode.primaryContentSections.first(where: { $0.kind == .content }) as? ContentRenderSection)
-            
-            XCTAssertEqual(Array(asides.content.dropFirst()), self.asidesStressTest)
+            let contentSection = try XCTUnwrap(renderNode.primaryContentSections.first(where: { $0.kind == .content }) as? ContentRenderSection)
+            let blockContent = contentSection.content.dropFirst()
+            let asides: [RenderBlockContent.Aside] = blockContent.compactMap { block in
+                guard case let .aside(aside) = block else {
+                    XCTFail("Unexpected block content in Asides.symbols.json")
+                    return nil
+                }
+                return aside
+            }
+            XCTAssertEqual(25, asides.count)
```

Comment:
Now that there is an `expectedAsides` variable, should this compare to that `expectedAsides.count`?